### PR TITLE
Add cache folders to Chrome and Firefox targets

### PR DESCRIPTION
**Pull request by nemesislvlup**

### CODE REVIEW

**Comment by EricZimmerman** [comment]
why do we want cache in with the main one? cache was left out to remove a lot of noise and small garbage files.

can you make a separate target called something like

BrowserCacche.tkape

with the sole purpose of pulling cache for all the browsers? that way it stays modular and people can include/exclude as they see fit

File: Targets/Browsers/Chrome.tkape [hunk 1]
```diff
@@ -154,6 +154,11 @@ Targets:
         Category: Communications
         Path: C:\users\%user%\AppData\Local\Google\Chrome\User Data\*\
         FileMask: Web Data*
+    -
+        Name: Chrome Cache Folder
+        Category: Communications
+        Path: C:\users\%user%\AppData\Local\Google\Chrome\User Data\Default\Cache\
```
Comment (Jil) [inline Targets/Browsers/Chrome.tkape:160]: Needs to support multiple profiles
```suggestion
        Path: C:\users\%user%\AppData\Local\Google\Chrome\User Data\*\Cache\
```

**Review by Jil — CHANGES_REQUESTED** [review]
The Chrome path is missing multiple profiles support

**Comment by nemesislvlup** [comment]
> why do we want cache in with the main one? cache was left out to remove a lot of noise and small garbage files.
> 
> can you make a separate target called something like
> 
> BrowserCacche.tkape
> 
> with the sole purpose of pulling cache for all the browsers? that way it stays modular and people can include/exclude as they see fit

Well, the IE and Edge targets do include the caches...

**Comment by nemesislvlup** [comment]
Added a new pull request that removes the Caches from Edge/IE tkapes and adds them for all browsers into a new tkape.

**Comment by EricZimmerman** [comment]
Thanks for doing that. i think it will be a lot nicer to have the caches separate as it gives more targeted collection capabilities.

**Pull request merged**